### PR TITLE
add hard dataset

### PR DESCRIPTION
**Pull request by zaidalyafeai**
Hotel Reviews in Arabic language.

### CODE REVIEW

**Review by sumanthd17 — COMMENTED** [review]
run `make style` and `flake8 datasets`

that should solve the code quality CI pipeline issue

**Comment by zaidalyafeai** [comment]
Thanks @sumanthd17 that fixed it.

File: datasets/hard/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+annotations_creators: []
+language_creators: []
```
Comment (lhoestq) [inline datasets/hard/README.md:3]:
```suggestion
annotations_creators:
- found
language_creators:
- found
```

File: datasets/hard/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+languages:
+- ar
+licenses: []
```
Comment (lhoestq) [inline datasets/hard/README.md:6]:
```suggestion
licenses:
- unknown
```

File: datasets/hard/README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 10k<n<100k
+source_datasets: []
```
Comment (lhoestq) [inline datasets/hard/README.md:11]:
```suggestion
source_datasets:
- original
```

**Review by lhoestq — APPROVED** [review]
Really cool ! Thanks for adding this one

**Pull request merged**